jest.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = async () => ({
     "!**/vendor/**",
     "!<rootDir>/assets/src/scripts/outputs-viewer/tests/*",
     "!<rootDir>/assets/src/scripts/outputs-viewer/index.jsx",
+    "!<rootDir>/assets/src/scripts/outputs-viewer/context/FilesProvider.jsx",
```

Comment:
Is this because we're not testing the code in FilesProvider directly?

Reply:
Yes, this file just sets up the context provider in the way recommended by React. It would be largely pointless to test when the code is unlikely to change, and the tests don't rely on it.